pkg/api/validation/validation.go

```diff
@@ -444,6 +444,10 @@ func validateVolumeSource(source *api.VolumeSource, fldPath *field.Path) field.E
 			numVolumes++
 			allErrs = append(allErrs, validateISCSIVolumeSource(source.ISCSI, fldPath.Child("iscsi"))...)
 		}
```

Comment:
@liggitt
I understand your concern especially from security perspective.
In order to validate initiatorname, we should research what kind of values should be accepted to avoid degradation.

@rootfs 
Any consideration?